demos/esc/prj.conf

```diff
@@ -0,0 +1,39 @@
+# 
+# This file is part of the efc project <https://github.com/eurus-project/efc/>.
+# Copyright (c) 2024 The efc developers.
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3.
+# 
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+CONFIG_CONSOLE=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_PWM=y
+CONFIG_PWM_STM32=y
+
+CONFIG_DEBUG=y
+
+CONFIG_STM32_FLASH_PREFETCH=y
+CONFIG_FPU=y
+
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_LOG=y
+CONFIG_DEBUG_COREDUMP=y
+CONFIG_DEBUG_COREDUMP_BACKEND_LOGGING=y
+
+CONFIG_CBPRINTF_FP_SUPPORT=y
+
+CONFIG_INPUT=y
```

Comment:
Some of these Kconfig settings are not needed for the demo, for instance this one.